libraries/AP_Logger/AP_Logger_File.cpp

```diff
@@ -310,8 +302,8 @@ void AP_Logger_File::Prep_MinSpace()
             break;
         }
         if (file_exists(filename_to_remove)) {
-            hal.console->printf("Removing (%s) for minimum-space requirements (%.2f%% < %.0f%%)\n",
-                                filename_to_remove, (double)avail, (double)min_avail_space_percent);
+            hal.console->printf("Removing (%s) for minimum-space requirements (%.0fMB < %.0fMB)\n",
+                                filename_to_remove, (double)avail*0.001f, (double)target_free*0.01f);
```

Comment:
SI?  Really? :-P

Also, are we missing a zero here?

Reply:
added missing zero

Reply:
was actual missing 4 zeros